src/Hastlayer/Hast.Xilinx/Drivers/VitisDeviceDriverBase.cs

```diff
@@ -0,0 +1,31 @@
+using Hast.Layer;
+using Hast.Synthesis.Models;
+using Hast.Synthesis.Services;
+using Hast.Xilinx.Helpers;
+using System;
+using static Hast.Common.Constants.DataSize;
+using static Hast.Common.Constants.Frequency;
+
+namespace Hast.Xilinx.Drivers;
+
+public abstract class VitisDeviceDriverBase : DeviceDriverBase
+{
+    public abstract string PlatformName { get; }
+    public abstract uint ClockFrequencyMhz { get; }
+
+    protected VitisDeviceDriverBase(ITimingReportParser timingReportParser)
+        : base(timingReportParser) =>
+        _deviceManifest = new Lazy<IDeviceManifest>(() => new VitisDeviceManifest
+        {
+            Name = DeviceName,
+            ClockFrequencyHz = ClockFrequencyMhz * Mhz,
+            SupportedCommunicationChannelNames = new[] { Constants.VitisCommunicationChannelName },
+            // While there is 64GB DDR RAM the max object size in .NET is 2GB. So until we add paging to SimpleMemory
```

Comment:
This refers to a specific card, not all Vitis ones have 64GB RAM.

Reply:
The exact same comment was everywhere. Ok, I've made it more vague.